src/pages/index.js

```diff
@@ -70,24 +70,52 @@ const IndexPage = () => {
     textColor: COLOR_CLAVE_SKIN,
   };
 
+  const scoverlayElement = useRef();
+
+  useLayoutEffect(() => {
+    const setClipPath = () => {
+      scoverlayElement.current.style.clipPath = window.innerWidth > window.innerHeight ?
+        `circle(calc(${1.5 * window.scrollY}px + min(45vh, 45vw)) at 85% 70%)` :
+        `circle(calc(${1 * window.scrollY}px + 20vh) at 85% 90%)`
+    }
+    setClipPath();
```

Comment:
så initiell clipPath, deretter endres den ved scroll?

Reply:
Korrekt. Så linje 81 setter initiell clipPath til et høvelig pent sirkel-utsnitt ved load og så oppdateres den ved scroll. Satt først initiell clipPath i stylingen (linje 211) som helgardering, men det ble litt for mye repetisjon når man skal ta hensyn til flere typer skjermer.